web/modules/custom/joinup_debug/joinup_debug.info.yml

```diff
@@ -0,0 +1,5 @@
+name: 'Joinup debug'
+type: module
+description: 'Debug module for the Joinup platform.'
```

Comment:
I was thinking that this `joinup_debug` module could be used for possible future issues as well, but since this is a very rare case, I will change it accordingly.